scipy/stats/_entropy.py

```diff
@@ -144,8 +144,21 @@ def entropy(pk: np.typing.ArrayLike,
         S /= np.log(base)
     return S
 
+
+def _differential_entropy_is_too_small(samples, kwargs):
+    values = samples[0]
```

Comment:
It looks like `samples` may not be guaranteed to be an array yet?

Reply:
The `np.split` call above each call to the `is_too_small` function will convert all the samples to numpy arrays.

Reply:
Oops, I was looking at where `is_too_small` is defined, not where it is called.